articles/azure-large-instances/workloads/epic/onboarding-requirements.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Onboarding requirements
+description: Provides an overview of onboarding requirements for ALI for Epic.
+ms.topic: conceptual
+author: jjaygbay1
+ms.author: jacobjaygbay
+ms.service: baremetal-infrastructure
+ms.date: 06/01/2023
+---
+
+# ALI for Epic onboarding requirements
+
+This article suggests actions to take after you receive an environment from the Microsoft ALI team.
+
+## Azure portal
+
+* Use Azure portal to:
+  * Create Azure Virtual Network (or networks) and ExpressRoute Gateway or Gateways with High or Ultra Performance Reference.
+  * Link them with ALI for Epic stamps using the Circuit/peer ID and Authorization Keys provided by Microsoft team.  
+
+## VNET address space
+
+Ensure that the VNET address space provided in your request is the same as what you configure.  
+
+## Time sync
+
+Setup time synchronization with NTP server.  
+
+## Jump box
+
+* Set up a jump box in a VM to connect to ALI for Epic stamps.
+* Change the root password at first login and store password in a secure location.  
+
+## Satellite server
+
+Install a red hat satellite server in a VM for RHEL 8.4 and patch download.
+
+## ALI for Epic stamps
+
+* Validate ALI for Epic stamps and configure and patch OS based on your requirements.  
```

Comment:
should be Validate ALI for Epic **servers**

Reply:
@vasthomsft wanted to verify the title on line 38 is accurate with, ALI for Epic **stamps**... yes?